storage/postgreskv/schema/migrate.go

```diff
@@ -7,22 +7,38 @@ package schema
 
 import (
 	"database/sql"
+	"strconv"
 
 	"github.com/golang-migrate/migrate/v3"
 	"github.com/golang-migrate/migrate/v3/database/postgres"
 	bindata "github.com/golang-migrate/migrate/v3/source/go_bindata"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/dbutil/pgutil"
 )
 
 // PrepareDB applies schema migrations as necessary to the given database to
 // get it up to date.
-func PrepareDB(db *sql.DB) error {
+func PrepareDB(db *sql.DB, dbURL string) error {
 	srcDriver, err := bindata.WithInstance(bindata.Resource(AssetNames(),
 		func(name string) ([]byte, error) {
 			return Asset(name)
 		}))
 	if err != nil {
 		return err
 	}
+
+	schema, err := pgutil.ParseSchemaFromConnstr(dbURL)
+	if err != nil {
+		return errs.New("error parsing schema: %+v", err)
+	}
+	if schema != "" {
+		err := createSchema(db, schema)
```

Comment:
There is `pgutil.CreateSchema`

Reply:
oops you are right, i missed that. thanks